src/autoyast-rnc/bootloader.rnc

```diff
@@ -66,6 +66,7 @@ bl_global =
       element os_prober       { STRING_BOOL }? &
       element suse_btrfs      { STRING_BOOL }? &
       element secure_boot      { STRING_BOOL }? &
+      element update_nvram      { STRING_BOOL }? &
```

Comment:
Do not forget to update the yast2-schema dependency.

https://github.com/yast/yast-schema/blob/master/package/yast2-schema.spec#L48